pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-7735-family-alias.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="lkapanad" id="es-hide-family-alias">
+        <createTable tableName="es_hidden_alias">
+            <column name="study_guid" type="varchar(10)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="alias" type="varchar(200)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="stable_id" type="varchar(45)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addUniqueConstraint columnNames="study_guid, stable_id" tableName="es_hidden_alias"/>
+
+        <insert tableName="es_hidden_alias">
```

Comment:
@LKapanad 
Looks good one minor renaming: Can we please rename table name to study_data_alias rather than es_hidden_alias. we don't need it be elastic specific. We are hiding data from CSV upload too and in future it might be something else.
Please rename the Dao as well to reflect table name.
May be I should have suggested name yesterday, sorry for additional refactor